backend/core/models.py

```
@@ -717,27 +683,176 @@
+            ("annotation", self.new_library.annotation),
+            ("translations", self.new_library.translations),
+            ("copyright", self.new_library.copyright),
+            ("objects_meta", self.new_library.objects_meta),
+        ]:
+            setattr(self.old_library, key, value)
+        self.old_library.save()
+
+        for new_dependency in new_dependencies:
+            self.old_library.dependencies.add(new_dependency)
+
+        i18n_object_dict = {
```

Ruff (0.11.9)

Local variable `i18n_object_dict` is assigned to but never used

Remove assignment to unused variable `i18n_object_dict`

(F841)